MobileCenterCrashes/MobileCenterCrashes/Internals/MSCrashHandlerSetupDelegate.h

```diff
@@ -0,0 +1,32 @@
+#import <Foundation/Foundation.h>
+
+/**
+ * This is required for Wrapper SDKs that need to provide custom behavior 
+ * surrounding the setup of crash handlers.
+ */
+@protocol MSCrashHandlerSetupDelegate <NSObject>
```

Comment:
That is intentional, I mean the word "setup" as a noun, not the verb phrase "set up"